src/main/java/ch/njol/skript/conditions/CondArmorStandBehaviour.java

```diff
@@ -0,0 +1,54 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Keywords;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.ArmorStand;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Armor Stand - Has Behaviour")
+@Description("Allows users to check the behaviour of an armor stand (i.e. whether it is ticking or able to move).")
+@Examples({
+	"if {_armorstands::*} are not able to tick:",
+	"if {_armorstand} is able to move:",
+	"if {_armorstand} can tick",
+	"if {_armorstand} cannot move"
+})
+@Since("INSERT VERSION")
+@Keywords({"move", "moving", "ticking"})
+public class CondArmorStandBehaviour extends PropertyCondition<LivingEntity> {
+
+	static {
+		if (Skript.methodExists(ArmorStand.class, "canTick")) {
```

Comment:
I would recommend adding the RequiredPlugins annotation with which version this is available on